backend/code/archipelag/message/views.py

```diff
@@ -19,7 +21,26 @@ def message_create(request, market_id):
             form = MessageForm()
             service_name = event.type.service
             msg_for_user = "Twoja wiadomość na {} została dodana, dodaj wiadomości na inne platformy".format(service_name)
+            add_point_if_rules_allow(request.user.ngouser, market_id)
             return render(request, 'registration/message.html', {'form': form, 'message':msg_for_user})
     else:
         form = MessageForm()
-    return render(request, 'registration/message.html', {'form': form})
\ No newline at end of file
+    return render(request, 'registration/message.html', {'form': form})
+
+
+def add_point_if_rules_allow(ngo, market_id):
+    from archipelag.ngo.models import NgoUser
```

Comment:
why?

Reply:
problems with circling import

Reply:
models.py Cannot have imports from views.py (because View controls Models). Could you fix this problem?

Reply:
deleted, I don't know why I have this problem before